LICENSE-APACHE

```diff
@@ -186,7 +186,7 @@ APPENDIX: How to apply the Apache License to your work.
    same "printed page" as the copyright notice for easier
    identification within third-party archives.
 
-Copyright 2019 Ian McIntyre
+Copyright [yyyy] [name of copyright owner]
```

Comment:
This seems like a possible mistake

Reply:
This one looks OK. I incorrectly filled in the license boilerplate within the license, which doesn't follow the license's recommendation. We're expected to replace the *after* we copy the notice to the top of the source file.

Reply:
Ahhh